backend/libraries/utils/src/text_validation.rs

```diff
@@ -90,13 +90,23 @@ pub fn validate_group_name(name: &str, is_public: bool, subtype: Option<&GroupSu
                 Ok(())
             }
         }
-        Err(StringLengthValidationError::TooShort(f)) => Err(NameValidationError::TooShort(f)),
-        Err(StringLengthValidationError::TooLong(f)) => Err(NameValidationError::TooLong(f)),
+        Err(StringLengthValidationError::TooShort(s)) => Err(NameValidationError::TooShort(s)),
+        Err(StringLengthValidationError::TooLong(l)) => Err(NameValidationError::TooLong(l)),
     }
 }
 
-pub fn validate_user_group_name(name: &str) -> Result<(), StringLengthValidationError> {
-    validate_string_length(name, MIN_USER_GROUP_NAME_LENGTH, MAX_USER_GROUP_NAME_LENGTH)
+pub fn validate_user_group_name(name: &str) -> Result<(), UsernameValidationError> {
+    match validate_string_length(name, MIN_USER_GROUP_NAME_LENGTH, MAX_USER_GROUP_NAME_LENGTH) {
+        Ok(()) => {
+            if name.contains(' ') {
```

Comment:
Should be looking for any whitespace, not just space.

Reply:
name.contains(|c: char| c.is_ascii_whitespace())

Reply:
Fixed